src/lib/SDK.ts

```diff
@@ -1,163 +0,0 @@
-import { ClientConnection, MC_EVENTS } from 'message-event-channel';
-import { Frame } from './Frame';
-import { CONTEXT } from './Events';
-import { ERRORS_INIT } from './Errors';
-import { MediaLink } from './MediaLink';
-import { ContentLink } from './ContentLink';
-import { ContentItem } from './ContentItem';
-import { ContentReference } from './ContentReference';
-import { LocalesModel } from './models/Locales';
-import { Field, FieldSchema } from './Field';
-import { Form } from './Form';
-import { HttpClient } from './HttpClient';
-
-export interface IClientConnection extends ClientConnection {}
-/**
- * Expected format for the provided options
- */
-export interface OptionsObject {
-  window?: Window;
-  connectionTimeout?: number | boolean;
-  timeout?: number | boolean;
-  debug?: boolean;
-}
-export interface Options {
```

Comment:
Migration need to be update to explain what has happened to these exported interfaces